src/components/app.js

```diff
@@ -1,9 +1,25 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
 import '../assets/css/app.scss';
+import About from './about';
+import Contact from './contact';
+import Footer from './footer';
+import Header from './header';
+import Home from './home';
+import Services from './services';
 
 const App = () => (
-    <div>
-        <h1>Sweet Corner</h1>
+    <div className="app">
+        <div className="container">
+            <Header />
+
+            <Route path="/about" component={About} />
+            <Route path="/contact" component={Contact} />
+            <Route exact path="/" component={Home} />
```

Comment:
Nice use of routing here! The most important thing to note on this line is the use of the `exact` JSX attribute. By sending forward this attribute, we are telling React Router that the path for this component needs to match the current url EXACTLY, or it is not a match. To put it more clearly, this Route is being used for the `/` path, which is the pure domain name without any additional path. However, if there were an additional path, that `/` would still be present in the url, and this path would match. To address this, we say that the Home component should be loaded only if the path is `/` and NOTHING more. Good job on this!!